corehq/apps/domain/management/commands/make_superuser.py

```diff
@@ -27,6 +28,9 @@ def get_password_from_user():
 
     @signalcommand
     def handle(self, username, **options):
+        if settings.IS_SAAS_ENVIRONMENT:
```

Comment:
@dannyroberts Wondering if we need it for India too?

Reply:
We do need it for India too. May need to introduce a new setting if there's not already a setting that's set to True for staging, india, and production and False to most other envs (including dev).

Reply:
I have included `IS_INDIA_ENVIRONMENT`. I was not sure what to call `production`, `staging` and `India` together.